components/textfield/index.css

```diff
@@ -12,56 +12,74 @@
  */
 
 .spectrum-Textfield {
```

Comment:
Most of this top selector is reorganization, with some light renaming, changing of values, plus removal of the quiet variant.